extensions/measurement-tracking/src/contexts/TrackedMeasurementsContext/_hydrateStructuredReport.js

```diff
@@ -119,6 +120,10 @@ export default function _hydrateStructuredReport(
     }
   }
 
+  // When Hydrating the structure report we need to cleanup the cornerstone state
+  // since it will result to a duplicated states
+  cornerstone3DTools.annotation.state.removeAllAnnotations();
```

Comment:
Is this true? Where we doing this before? What if I want to fuse two SRs together? Didn't we used to have a workflow which was like "choose if you want to add these measurements to the current read, or start a new set"? cc @swederik